pkg/cmd/dotgit/contains.go

```diff
@@ -0,0 +1,28 @@
+package dotgit
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/MaineK00n/vuls-data-update/pkg/dotgit/contains"
+)
+
+func newCmdContains() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "contains <dotgit repository> <commit hash>",
+		Short: "Check if the commit hash is contained in the dotgit repository",
+		Example: heredoc.Doc(`
+			$ vuls-data-update dotgit contains ~/.cache/vuls-data-update/dotgit/vuls-data-raw-debian-security-tracker 729c12ba5ff2dafacaf26b9311d8dcbea1d98bd3
```

Comment:
Full hash string needed? It may be preferrable if short hash can be accepted, my two cents.

Reply:
Changed my mind. Specifying full hashes is good if this is used as API or by some programs/codes/scripts.